ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -620,7 +631,8 @@ int  EXTIO_API ExtIoSetSrate(int srate_idx)
 		giExtSrateIdx = srate_idx;
 		gExtSampleRate = (unsigned)(newSrate + 0.5);      // update
 
-		r2iqCntrl.Updt_SR_LO_TUNE(srate_idx, &glLOfreq, &glTunefreq);
+		r2iqCntrl.Setdecimate(4 - srate_idx); // reverse order 32,16,8,4,2 MHz
```

Comment:
I think this is not necessary. but I can be wrong and need to test

Reply:
I left SetDecimate and tested it ok.